examples/games/contributors.rs

```diff
@@ -120,12 +120,7 @@ fn setup_contributor_selection(mut commands: Commands, asset_server: Res<AssetSe
                 SpriteBundle {
                     sprite: Sprite {
                         custom_size: Some(Vec2::new(1.0, 1.0) * SPRITE_SIZE),
-                        color: LegacyColor::hsla(
-                            hue,
-                            SATURATION_DESELECTED,
-                            LIGHTNESS_DESELECTED,
-                            ALPHA,
-                        ),
+                        color: Color::hsla(hue, SATURATION_DESELECTED, LIGHTNESS_DESELECTED, ALPHA),
```

Comment:
Nit for future PR: This looks gross. I think it would look nicer to declare `const: SELECTED: Hsla = Hsla::new(0., 0.3, 0.2, 0.92);` and then use `SELECTED.with_hue(hue).into()`.

Reply:
I had some notes on other ideas to improve that example. I'll add this to the pile.